reference-implementation/to-upstream-wpts/transform-stream/errors.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+if (self.importScripts) {
+  self.importScripts('/resources/testharness.js');
+}
+
+const thrownError = new Error('bad things are happening!');
+thrownError.name = 'error1';
+
+promise_test(t => {
+  const results = [];
+
+  const ts = new TransformStream({
+    transform() {
+      throw thrownError;
+    }
+  });
+
+  const reader = ts.readable.getReader();
+
+  results.push(promise_rejects(t, thrownError, reader.read(),
```

Comment:
I agree with @domenic that the |results| array doesn't really help with readability.